mobile/mysterium/wireguard_connection_setup.go

```diff
@@ -157,20 +157,27 @@ func newTunnDevice(wgTunnSetup WireguardTunnelSetup, config wireguard.ServiceCon
 	return tunDevice, err
 }
 
-func waitUDPPortReadyFor(port int, timeout time.Duration) error {
+func waitUDPPortReadyFor(port int, timeout, checkInterval time.Duration) error {
```

Comment:
I'd return a cancel func here as well, to allow for early exit to the caller. as currently this might be stuck for quite a bit if I want to exit node.